pkg/synopsysctl/cmd_root.go

```diff
@@ -112,7 +112,7 @@ func init() {
 
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&kubeConfigPath, "kubeconfig", kubeConfigPath, "Path to a kubeconfig file with the context set to a cluster for synopsysctl to access")
-	rootCmd.PersistentFlags().BoolVar(&insecureSkipTLSVerify, "insecure-skip-tls-verify", insecureSkipTLSVerify, "Server's certificate won't be validated. HTTPS will be less secure")
+	rootCmd.PersistentFlags().BoolVar(&insecureSkipTLSVerify, "insecure-skip-tls-verify", insecureSkipTLSVerify, "Server's certificate won't be validated. HTTPS will be irrelevant")
```

Comment:
Saying that HTTPS will be irrelevant is not entirely true.
The traffic will still be encrypted but we will just not check that it’s coming from a trusted source